src/System.Runtime/tests/System/IntPtrTests.cs

```diff
@@ -164,7 +164,7 @@ private static void VerifyPointer(IntPtr ptr, long expected)
             Assert.Equal(expected32, ptr.ToInt32());
 
             Assert.Equal(expected.ToString(), ptr.ToString());
-            Assert.Equal(expected.ToString("x"), ptr.ToString("x"));
+            Assert.Equal(expected.ToString("x"), ptr.ToInt64().ToString("x"));
```

Comment:
This is not testing `IntPtr.ToString("x")` that it was meant to test. Should the expected value be changed instead?